### ISSUE TITLE
WIP: Security 3.24 Tracking issue Sprint 2

### ISSUE THREAD

**Issue by ElizabethStirling** [sourcegraph/sourcegraph#17031]
### Plan
Goals:
1. Continue to onboard André #16582 
1. Tie up loose ends WRT GCP centralized logging #15444, #15447, [Team Goal #1](https://about.sourcegraph.com/handbook/engineering/security/goals#visibility-into-sourcegraph-cloud-s-attack-surface)
1. Resolve known security issues [Team Goal #2](https://about.sourcegraph.com/handbook/engineering/security/goals#confidence-in-our-security-model)

@chayim please review these goals and update as relevant

### Availability
Sprint is from 2021-01-03 to 2021-01-22
Duration is 15 days, 16 for Chayim

Chayim: 16 days - no vacation
Elizabeth: 14 days  - local holiday 2020-01-18
André: 15 days
### Tracked issues

@ElizabethStirling: __10.00d__

- [ ] Team Security Tracking: Security Event Logging ([#15444](https://github.com/sourcegraph/sourcegraph/issues/15444)) __10.00d__
  - [ ] Make basic Kibana queries to help dig through logs ([#17053](https://github.com/sourcegraph/sourcegraph/issues/17053)) __3.00d__
  - [ ] Make basic Kibana Security Dashboard(s) to monitor the results of relevant security queries ([#17055](https://github.com/sourcegraph/sourcegraph/issues/17055)) __4.00d__
  - [x] (🏁 6 days ago) Centralize OS logs in centralized logging ([~#15447~](https://github.com/sourcegraph/sourcegraph/issues/15447)) __3.00d__

Completed: __3.00d__
- [x] (🏁 6 days ago) Centralize OS logs in centralized logging ([~#15447~](https://github.com/sourcegraph/sourcegraph/issues/15447)) __3.00d__

@andreeleuterio

- [ ] Andre Eleuterio: On-boarding ([#16582](https://github.com/sourcegraph/sourcegraph/issues/16582))

@chayim: __3.00d__

- [ ] Team Security Tracking: Security Event Logging ([#15444](https://github.com/sourcegraph/sourcegraph/issues/15444)) __3.00d__
  - [ ] Make basic Kibana queries to help dig through logs ([#17053](https://github.com/sourcegraph/sourcegraph/issues/17053)) __3.00d__

Completed
- [x] (🏁 1 day ago) Logs overwhelm centralized logging ([~#16974~](https://github.com/sourcegraph/sourcegraph/issues/16974))

### Security issues repo
@andreeleuterio
- [ ] Enforce SSL for dogfood bitbucket SSL connections ([security#123](https://github.com/sourcegraph/security-issues/issues/123))
- [ ] Stored XSS via javascript scheme through Quick links section ([security#94](https://github.com/sourcegraph/security-issues/issues/94))

#### Legend

- 👩 Customer issue
- 🐛 Bug
- 🧶 Technical debt
- 🎩 Quality of life
- 🛠️ [Roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.5nwl5fv52ess)
- 🕵️ [Spike](https://en.wikipedia.org/wiki/Spike_(software_development))
- 🔒 Security issue
- :shipit: Pull Request

**Comment by ElizabethStirling** [comment]
**Last Week**
Last week I worked on getting #15447 hammered out, but unfortunately wasn't able to get it merged in. I also got caught up on things I missed during my vacation.

**This Week**
This week, I've merged in #15447, and in the process found that Elastic has lit itself on fire. Currently working to extinguish that dumpster fire, after which I'll move on to #17053, and generate queries to start helping us sift through our log data.

**Comment by andreeleuterio** [comment]
**Last week**
Had several calls meeting folks from different teams which was really cool. Focused on https://github.com/sourcegraph/security-issues/issues/94 but was unable to wrap it up and found a bug that will be dealt with separately. Also planned next steps for https://github.com/sourcegraph/security-issues/issues/115 with other stakeholders.

**This week**
https://github.com/sourcegraph/security-issues/issues/94 needs a unit test and also getting approvals that might require rework.
https://github.com/sourcegraph/security-issues/issues/115 is all set to begin testing. I have a kickoff meeting with the code-intel team and hope begin testing on Wednesday.

**Comment by andreeleuterio** [comment]
**Last week**
Finally closed sourcegraph/security-issues#94. It took longer than expected because it required touching quite a bit of _old_ code. Also got started with sourcegraph/security-issues#115 and I have the access and context needed to test it.

**Next week**
Focus on sourcegraph/security-issues#115. I will finish testing and prepare a summary of what was tested to share with the team.

**Comment by ElizabethStirling** [comment]
**This Week**
This week I was thrown off balance by the Elastic cluster being in red status. After that, I had to reconfigure pubsubbeats and Elastic to handle the added workload from #15447. I'm not happy with how much this task slopped over into this week, but this expanded scope was critical for being able to move forwards. I've started #17053 and already generated some queries I'd like to monitor.

**Next Week**
Next week I want to continue learning Kibana's query language and digging through our data as part of #17053. After that, I'll get started on #17055, but don't expect to get to a finished state due to the aforementioned time sinks.
